### DISCUSSION TITLE
HB Blog Profile Sidebar Module

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#733] [General]
# HB Blog Profile Sidebar Module

Introduce yourself or your company with title, description, location, social media and so on.

https://hbstack.dev/modules/blog/sidebar/profile/

**Comment by diogoguilhen** [comment]
have some way to put the right size in  avatar_size ? like  avatar_size: 150x200?

**Reply by razonyang** [reply]
Fixed by `github.com/hbstack/blog/modules/sidebar/profile@v1.1.5`, now it will preserve the aspect ratio.

> Please note that the module resizes the avatar instead of cropping.

If you want to show full height of avatar, you can [tweak it via styles](https://hbstack.dev/docs/advanced/style/).

```
.hb-blog-sidebar-profile-img {
    border-radius: 0 !important;
    height: auto;
}
```

**Reply by diogoguilhen** [reply]
tks
